services/src/modules/validation.spec.ts

```diff
@@ -15,6 +15,7 @@ describe('validateMetadataCharacters - Validation that namespaces and names cont
       upstreams: [],
       upstreamClientCredentials: [],
       policies: [{ metadata: { namespace: 'other-ns', name: 'valid-chars_only' }, type: PolicyType.opa, code: 'code' }],
+      remoteSchemas: [],
```

Comment:
Also here, we probably shouldn't have to initialize it as empty if it's not a permanent part of the resource group